src/pages/create/EventForm.tsx

```diff
@@ -1,107 +1,34 @@
-import { yupResolver } from '@hookform/resolvers/yup';
-import { useRouter } from 'next/router';
-import { useMemo } from 'react';
-import { useForm } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
-import * as yup from 'yup';
 
-import type { Calendar } from '@/hooks/api/events';
-import type { StepsConfiguration } from '@/pages/Steps';
-import { Steps } from '@/pages/Steps';
-import { Page } from '@/ui/Page';
-import { Toast } from '@/ui/Toast';
-
-import { useToast } from '../manage/movies/useToast';
-import { AdditionalInformationStep } from '../steps/AdditionalInformationStep';
-import { CalendarStep } from './CalendarStep';
-import { ThemeStep } from './ThemeStep';
-import { TypeStep } from './TypeStep';
-
-type EventType = 'event' | 'place';
-
-type FormData = {
-  type: EventType;
-  theme: string;
-  calendar: Calendar;
-};
-
-const schema = yup
-  .object({
-    type: yup.mixed<EventType>().required().oneOf(['event', 'place']),
-    theme: yup.string().required(),
-    calendar: yup.mixed().required(),
-  })
-  .required();
+import { additionalInformationStepConfiguration } from '../steps/AdditionalInformationStep';
+import { StepsForm } from '../steps/StepsForm';
+import { typeStepConfiguration } from '../steps/TypeStep';
 
 const EventForm = () => {
```

Comment:
A general comment about most of the naming. Actually it's not the `EventForm` but a `OfferForm` since it can be an Event or Place. 

Functions like `convertFormDataToEvent ` would better be called `convertFormDataToOffer`